### ISSUE TITLE
normalize-css-color dependency missing

### ISSUE THREAD

**Issue by KannarFr** [gluestack/dank-style#71]
```bash
error: Error: Unable to resolve module normalize-css-color from .../@dank-style/react/src/utils/cssify/utils/normalizeColor.ts: normalize-css-color could not be found within the project or in these directories:
```

Looks it missing `normalize-css-color` as dependencies?

**Comment by surajahmed** [comment]
@KannarFr Thanks for reporting. We'll add `normalize-css-color` and  `inline-style-prefixer` as dependencies.

**Comment by munnokd** [comment]
Hey, @surajahmed  can you assign this issue to me?

**Comment by surajahmed** [comment]
@munnokd This issue is fixed in the latest version `0.3.16`. I appreciate your willingness to address the issue. Thanks.
